pkg/plugins/platform/local.go

```diff
@@ -94,7 +94,7 @@ func (l *LocalApp) Describe() (string, error) {
 	output = output + "\n\nOther Services\n--------------\n"
 	other := uitable.New()
 	other.AddRow("MailHog:", l.URL()+":8025")
-
+	other.AddRow("phpMyAdmin:", l.URL()+":8036")
```

Comment:
Could we move 8025 and 8036 to global variables (or some other generalization technique), so they could be changed globally if necessary? Using literals everywhere is not good in the long run.